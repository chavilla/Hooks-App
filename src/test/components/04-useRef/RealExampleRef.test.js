import React from 'react';
const { shallow } = require("enzyme")
const { RealExampleRef } = require("../../../components/04-useRef/RealExampleRef")

describe('RealExampleRef testing', () => {

    const wrapper=shallow(<RealExampleRef />);

    test('should the return the snapshot', () => {

      expect(wrapper).toMatchSnapshot();
      expect(wrapper.find('MultiCustomHook').exists()).toBe(false);

    });

    test('should to get a component', () => {
      
        const btn=wrapper.find('.btn');
      btn.simulate('click');
      expect(wrapper.find('MultiCustomHook').exists()).toBe(true);
    
    })
    
    
})