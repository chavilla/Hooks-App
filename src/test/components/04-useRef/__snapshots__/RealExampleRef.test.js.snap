// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RealExampleRef testing should the return the snapshot 1`] = `
<Fragment>
  <h2>
    Real Example Ref
  </h2>
  <hr />
  <button
    className="btn btn-primary mt-5"
    onClick={[Function]}
  >
    Ocultar/mostrar
  </button>
</Fragment>
`;
